sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcTest.scala

```diff
@@ -143,6 +143,13 @@ abstract class OrcTest extends QueryTest with FileBasedDataSourceTest with Befor
     spark.read.orc(file.getAbsolutePath)
   }
 
+  def withAllOrcReaders(code: => Unit): Unit = {
```

Comment:
Hi, All.
This naming is misleading because this only tests `native` ORC reader.
Apache Spark provides `hive` ORC reader, too.

Instead of `withAllOrcReaders`, let's use `withAllNativeOrcReaders`.